### DISCUSSION TITLE
Save a custom Input Map for use in other projects

### DISCUSSION THREAD

**Discussion by joshuapitaniello** [godotengine/godot-proposals#9344] [General Discussions]
Much like how users can save an editor layout for their own custom needs I feel this feature would be right at home with Godot. Since there is already a default input map and their is a default editor layout I think this feature could save people some time.

**Comment by AThousandShips** [comment]
Related:
* https://github.com/godotengine/godot-proposals/issues/423

Assuming you mean the map for the editor, otherwise this is a duplicate

**Reply by joshuapitaniello** [reply]
oh I looked for a duplicate and didn't find one. My apologies
